regression/cbmc/gcc_builtin_mul_overflow/main.c

```diff
@@ -0,0 +1,125 @@
+#include <assert.h>
+#include <limits.h>
+#include <stdint.h>
+
+#ifdef __CPROVER
+#  define assume(X) __CPROVER_assume(X)
+#else
+#  define assume(X)                                                            \
+    do                                                                         \
+    {                                                                          \
+    } while(0)
+#endif
```

Comment:
The `assume` macro doesn't seem to be used anywhere.

Reply:
Good point